### ISSUE TITLE
Plugin does not work

### ISSUE THREAD

**Issue by andrea06** [ev4debug/Leaflet.TimeDimension.WmsTimeInterval#1]
Hi,
I tried the plugin according readme doc, but it does not work, I get a type error: "this._timeDimension.getfixedStartTimeValue is not a function".

Andrea

**Comment by ev4debug** [comment]
Hi @andrea06 ,
I've found and hopefully fixed the bug in the **Leaflet.TimeDimension.WmsTimeInterval.js** file.
Now in map initialization instead of `timeDimension: true,` you must use `timeDimensionWmsTimeInterval: true,`.
I've also added a simple example that works only with CORS enabled (or by setting a proxy in the     L.timeDimension.layer.wms.wmsTimeInterval options).
In the next days I'll try to write a more useful readme an maybe some more examples.

Thanks!!! 

Marco
